src/StatsTools/ratio_of_means.jl

```diff
@@ -26,25 +26,36 @@ end
 # Base.iterate(r::RatioBlockingResult, args...) = iterate(r.ratio, args...)
 # Base.length(r::RatioBlockingResult) = length(r.ratio)
 # Base.eltype(r::RatioBlockingResult) = eltype(r.ratio)
-MacroTools.@forward RatioBlockingResult.ratio Statistics.median, Statistics.quantile
-MacroTools.@forward RatioBlockingResult.ratio Statistics.middle, Base.iterate, Base.extrema
-MacroTools.@forward RatioBlockingResult.ratio Base.minimum, Base.maximum
-MacroTools.@forward RatioBlockingResult.ratio Statistics.mean, Statistics.cov
-
-function Statistics.median(r::RatioBlockingResult{<:Complex})
-    complex(median(real(r.ratio)), median(imag(r.ratio)))
+MacroTools.@forward RatioBlockingResult.ratio MonteCarloMeasurements.pmedian, MonteCarloMeasurements.pquantile
+MacroTools.@forward RatioBlockingResult.ratio MonteCarloMeasurements.pmiddle, MonteCarloMeasurements.piterate, MonteCarloMeasurements.pextrema
+MacroTools.@forward RatioBlockingResult.ratio MonteCarloMeasurements.pminimum, MonteCarloMeasurements.pmaximum
+MacroTools.@forward RatioBlockingResult.ratio MonteCarloMeasurements.pmean, MonteCarloMeasurements.pcov
```

Comment:
Is there an easy way to document the `p*` functions? I.e. write a single docstring for all of them, maybe just mentioning the respective functions from `Base` and `Statistics`?

Also, when I think about it `piterate` seems really awkward...

Reply:
The only way is to add a docstring for each. Maybe the fact that you have to use `p*` functions should me mentioned somewhere clearly visible in the docs?

I must say I'm not a big fan `p*` functions in general. It seems much nicer than functions from `Base` and `Statistics` just work.